### ISSUE TITLE
Integrate Attestation-Service to verify TEE evidence

### ISSUE THREAD

**Issue by jialez0** [confidential-containers/kbs#18]
Generic KBS needs to integrate [`Attestation-Service`](https://github.com/confidential-containers/attestation-service), and then KBS can forward TEE evidence to it, and verify according to the owner's configured policy. We can start this work after #16 is completed.

Attestation Service can be integrated by KBS in two forms:

1. As a Rust library crate.
2. As a gRPC server application.

We should provide both above two options in the future. The first one can be used by users who want the KBS deployment to be as simple as possible, but may discard some functions of AS (such as monitoring the upstream supply chain to update reference values at any time). The second type will provide more complete functions, but there will be some additional operations in the deployment.

**Comment by jialez0** [comment]
At present, we have implemented the integration of AS in the form of rust library crute, so this issue can be closed. We will track the support progress of the independent AS application option in the new issue.
